catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -47,6 +47,10 @@ public abstract class JdbcTableOperations implements TableOperation {
   public static final String COMMENT = "COMMENT";
   public static final String SPACE = " ";
 
+  public static final String MODIFY_COLUMN = "MODIFY COLUMN ";
+
+  public static final String AFTER = "AFTER ";
+
```

Comment:
I would remove the extra blank lines here

Reply:
indeed done.